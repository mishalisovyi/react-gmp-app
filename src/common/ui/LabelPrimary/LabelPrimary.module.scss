@use '../../../mixins';
@use '../../../colors';

.LabelPrimary {
  @include mixins.typography-body;

  display: flex;
  justify-content: center;
  align-items: center;
  width: 60px;
  height: 60px;
  color: colors.$white;
  border: solid 1px colors.$white;
  border-radius: 50%;
}