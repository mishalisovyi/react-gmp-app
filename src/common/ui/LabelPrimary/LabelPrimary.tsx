import React from 'react';
import styles from './LabelPrimary.module.scss';

interface LabelPrimaryProps {
  value: string | number;
}

export function LabelPrimary({ value }: LabelPrimaryProps) {
  return <span className={styles['LabelPrimary']}>{value}</span>;
}
