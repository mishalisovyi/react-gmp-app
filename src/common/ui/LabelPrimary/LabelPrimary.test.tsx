import React from 'react';
import renderer from 'react-test-renderer';

import { LabelPrimary } from './LabelPrimary';

it('should render primary label', () => {
  const tree = renderer.create(<LabelPrimary value="test" />);

  expect(tree.toJSON()).toMatchSnapshot();
});
