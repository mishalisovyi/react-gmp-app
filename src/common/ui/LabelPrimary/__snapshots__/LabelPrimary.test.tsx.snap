// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render primary label 1`] = `
<span>
  test
</span>
`;
